docs/manage/fleet/reuse-configuration.md

```diff
@@ -401,16 +386,14 @@ Fragment overwrites are currently _not_ supported for modifying [triggers](/mana
 
```

Comment:
[out of diff] Do we still need to say "Fragment overwrites are currently not supported for modifying triggers" if triggers themselves aren't supported in fragments?